ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_core.cc

```diff
@@ -420,8 +420,12 @@ class TFBuffer : public TensorBuffer {
   size_t len_;
 
 #if (TF_MAJOR_VERSION == 1 && TF_MINOR_VERSION < 13)
+  explicit TFBuffer (void* data_ptr) : data_(data_ptr) {}
   void* data () const override { return data_; }
+#elif (TF_MAJOR_VERSION == 1 && TF_MINOR_VERSION == 13)
```

Comment:
wouldn't it be ok to say ```>=``` instead of ```==``` ?

Reply:
It'd be better if you handle TF_MAJOR_VERSION != 1
e.g., 
```
#else
#error This supports Tensorflow 1.x only.
```
until we build & test tensorflow 2.x w/ nnstreamer.

Reply:
> wouldn't it be ok to say >= 13 instead of == 13 ?

the change is maintained after 1.13. So I applied `>= 13`.

> It'd be better if you handle TF_MAJOR_VERSION != 1

it's applied